server/utils/viewUtils.ts

```diff
@@ -13,6 +13,14 @@ export default class ViewUtils {
     return escape(val).val
   }
 
+  static nl2br(val: string): string {
```

Comment:
The type of `val` is a bit confusing here given that on the next line we're checking whether it's a `string`. Would `unknown` be more appropriate?

Reply:
see above comments. we want this method to only be called with strings. but due to some weirdness it ends up getting called with other stuff too.